api/src/main/java/marquez/api/LineageResource.java

```diff
@@ -19,8 +19,13 @@
 import marquez.service.ServiceFactory;
 import marquez.service.models.NodeId;
 
+/**
+ * @deprecated This resource is no longer supported
+ *     <p>Use {@link OpenLineageResource#getLineage(NodeId, int)} instead.
+ */
 @Path("/api/v1-beta")
 @Slf4j
+@Deprecated(since = "0.16")
```

Comment:
_Minor:_ I would set _since_ to `0.16.0` to be specific.

Reply:
ok, will do it in next PR